engine/schema/src/com/cloud/upgrade/DatabaseUpgradeChecker.java

```diff
@@ -242,13 +243,13 @@ public DatabaseUpgradeChecker() {
 
         _upgradeMap.put("4.5.2", new DbUpgrade[] {new Upgrade452to460(), new Upgrade460to461(), new Upgrade461to470()});
 
-        _upgradeMap.put("4.5.3", new DbUpgrade[] {new Upgrade453to460()});
+        _upgradeMap.put("4.5.3", new DbUpgrade[] {new Upgrade453to460(), new Upgrade460to461(), new Upgrade461to470()});
 
         _upgradeMap.put("4.6.0", new DbUpgrade[] {new Upgrade460to461(), new Upgrade461to470()});
 
         _upgradeMap.put("4.6.1", new DbUpgrade[] {new Upgrade461to470()});
 
-        _upgradeMap.put("4.6.2", new DbUpgrade[] {new Upgrade461to470()});
+        _upgradeMap.put("4.6.2", new DbUpgrade[] {new Upgrade462to470()});
```

Comment:
The PR tries to make the usage be consistent with the implementation and usage; see Upgrade453to460 class for example. The fix here is mainly for the 4.5.3 version, 4.6.2 upgrade path is semantically broken while it still works is another thing.